dali/benchmark/video_reader_bench.cc

```diff
@@ -0,0 +1,159 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <benchmark/benchmark.h>
+#include <iostream>
+
+#include "dali/benchmark/dali_bench.h"
+#include "dali/pipeline/pipeline.h"
+#include "dali/core/common.h"
+
+#define GET_ENV(env_var, default_val) (std::getenv(env_var)) ? \
```

Comment:
```suggestion
#define GET_ENV_INT(env_var, default_val) (std::getenv(env_var)) ? \
```
?

Also, could this be a function instead of a macro?